e2e/pierced-react/functions/_middleware.ts

```diff
@@ -14,9 +14,8 @@ const getGatewayMiddleware: ((devMode: boolean) => PagesFunction) & {
         z-index: 9999999999999999999999999999999;
     
         &.remix {
-            bottom: 14%;
-            left: 50%;
-            translate: -50% 0;
+            bottom: 16%;
+            left: 15%;
```

Comment:
I think this is never going to work fine, the only solution is to add some scaffolding HTML 😕 
(at least I think)

(anyways I'm not against the change, just saying that I don't think it's going to change much)